client/src/pages/CreatePokemon.jsx

```diff
@@ -0,0 +1,253 @@
+     */
+    const randName = async () => {
+        // use some API to generate random pokemon name
+        const randName = await API.getPokemonName();
+        console.log(randName);
+        setName(Math.random());
+    };
+
+    return (
+        <div>
+            <Navbar />
+            <form onSubmit={handleCreatePokemon} className="create-page">
+                <div className="drawing">
+                    <h2>Draw your Pokemon here!</h2>
+                    <div className="canvas">
+                        <CanvasDraw hideGrid={true}
+                            hideInterface={true}
+                            brushColor={color}
+                            brushRadius={5}
+                            lazyRadius={0}
+                            ref={canvasInput}
+                            style={{ width: '100%', height: '100%' }} />
+                    </div>
+                    <div className="color-picker">
+                        <label htmlFor="select-color">Color</label>
+                        <input type="color" id="select-color" value={color} onChange={(e) => setColor(e.target.value)}></input>
+                    </div>
+                </div>
+                <div className="pokemon-form">
+                    <h2>Create a Pokemon</h2>
+                    <div className="name-row">
+                        <label htmlFor="name">Name</label>
+                        <input type="text" id="name" name="name" defaultValue={name} required></input>
+                        <button type="button" onClick={randName}>Generate Random Name</button>
+                    </div>
+                    <div className="desc-row">
+                        <label htmlFor="desc">Description</label>
+                        <input type="text" id="desc" name="desc" required></input>
+                    </div>
+                    <div className="type-row">
+                        <label htmlFor="type1">Type 1</label>
+                        <select id="type1" name="type1">
+                            <option>Normal</option>
+                            <option>Fire</option>
+                            <option>Water</option>
+                            <option>Grass</option>
+                            <option>Electric</option>
+                            <option>Psychic</option>
+                            <option>Ice</option>
+                            <option>Dragon</option>
+                            <option>Dark</option>
+                            <option>Fairy</option>
+                            <option>Fighting</option>
+                            <option>Flying</option>
+                            <option>Poison</option>
+                            <option>Ground</option>
+                            <option>Rock</option>
+                            <option>Bug</option>
+                            <option>Ghost</option>
+                            <option>Steel</option>
+                        </select>
```

Comment:
if you can, try to make this a separate component since it's being re-used a bunch of times in the same page